src/custom/theme/styled.d.ts

```diff
@@ -80,6 +80,14 @@ declare module 'styled-components' {
       borderRadius?: string
       boxShadow?: string
     }
+    buttonSecondary: {
```

Comment:
seconded :point_up:

Reply:
In my view the 'Wrap ETH' was a separate special action.

Ideally for a direct Primary <-> Secondary button relation, we'd have one use the primary (orange) color and the other one blank. But for lack of a design system this might not be clear why for wrap ETH a different color was used.

Let's address this when I try to create some mini design system so we have some overview.